src/components/Heading/Heading.module.css

```diff
@@ -0,0 +1,31 @@
+.santa {
```

Comment:
With css modules you should be able to place santa_headline inside santa style (and call it only headline), try to refactor to nesting css classes